src/DssSpell.sol

```diff
@@ -106,226 +104,304 @@ contract DssSpellAction is DssAction {
     //
     // uint256 internal constant X_PCT_RATE      = ;
```

Comment:
BTW rates are still below the constants

Rates go below the example PCT rate thing (this) and then constants (MILLION etc) go below that